changelogs/current.yaml

```diff
@@ -32,6 +32,10 @@ minor_behavior_changes:
 - area: rate_limit
   change: |
     add ``MONTH`` and ``YEAR`` to the unit of time for rate limit.
+- area: stats
+  change: |
+    Properly extract the ext_authz stat-prefix as a tag when exporting prometheus metrics.  This behavior can be reverted by setting runtime guard ``envoy.restart_features.ext_authz_prefix_handling`` to false.
```

Comment:
@alyssawilk 
Should we be default this to false initially first? I haven't yet read the new policy :)